src/amo/components/Home/styles.scss

```diff
@@ -0,0 +1,195 @@
+@import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
+
+.Home {
+  padding: $padding-page;
+}
+
+.Home-category-card {
+  margin-bottom: 10px;
+
+  .Card-contents {
+    @include respond-to(medium) {
+      display: flex;
+      flex-flow: row nowrap;
+    }
+  }
+}
+
+.Home-text-wrapper {
+  align-self: auto;
+
+  @include respond-to(medium) {
+    align-self: center;
+  }
+
+  @include respond-to(large) {
+    max-width: 200px;
+  }
+}
+
+.Home-subheading {
+  line-height: 1.2;
+  margin-bottom: -10px;
+  margin-top: 18px;
+  text-align: center;
+
+  @include respond-to(medium) {
+    @include text-align-start();
+  }
+}
+
+.Home-description {
+  display: none;
+
+  @include respond-to(medium) {
+    display: block;
+  }
+}
+
+.Home-category-list {
+  display: flex;
+  flex-flow: row wrap;
+  overflow: auto;
+  margin-bottom: 0;
+  padding: 0;
+  width: 100%;
+
+  @include respond-to(large) {
+    flex-wrap: nowrap;
+  }
+}
+
+.Home-category-li {
+  background-color: $base-color;
+  background-size: 50% auto;
+  background-position: 50% 35%;
+  background-repeat: no-repeat;
+  border-radius: 6px;
+  display: block;
+  flex-grow: 1;
+  list-style-type: none;
+  margin: 5px;
+  overflow: auto;
+  padding: 0;
+  position: relative;
+  text-align: center;
+  width: 25%;
+
+  a:link {
+    font-size: 10px;
+    text-decoration: none;
+    padding-top: 70%;
+    display: block;
+  }
+
+  span {
+    display: block;
+    padding: 5px 10px;
+    text-align: center;
+    text-decoration: inherit;
+  }
+
+  a:hover,
+  a:focus {
+    text-decoration: underline;
+  }
+}
+
+.Home-category-link,
+.Home-extensions-link,
+.Home-themes-link {
+  &,
+  &:visited {
+    color: $link-color;
+  }
+}
+
+.Home-extensions-link:link,
+.Home-themes-link:link {
+  align-items: center;
+  background: $base-color;
+  border-radius: 6px;
+  display: flex;
+  font-size: 10px;
+  justify-content: center;
+  margin: 0 auto 20px;
+  padding: 10px;
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  width: calc(100% - 40px);
+
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+
+  &::before {
+    content: '';
+    display: inline-block;
+    width: 16px;
+    height: 16px;
+    margin: 0 5px;
+  }
+}
+
+.Home-extensions-link:link::before {
```

Comment:
Yeah I really forget why we're using `:link` on `a` styles but my goodness is it frustrating.